src/TypeLevel/RowList/Unordered.purs

```diff
@@ -0,0 +1,34 @@
+module TypeLevel.RowList.Unordered where
+
+import TypeLevel.Nat
+import Type.RowList as RL
+import Prim.TypeError (class Fail, Text)
+import Data.Symbol
+
+-- | A kind for UNORDERED RowLists. Actual RowLists are automagically ordered lexicographically, Plutus Data is not ordered
+-- lexicographically, so we need this to encode the structure of Plutus Data.
+data RowList :: forall (k :: Type). k -> Type
+data RowList k
```

Comment:
Doesn't RowList maintain ordering? Afaict it's just *RowToList*, and row types themselves that maintain the lexicographic ordering

Reply:
I swear I read something (in a blog post maybe?) about RowList maintaining lexicographic ordering but looking through the module documentation that doesn't seem to be the case. Perhaps what I read was mistaken? I'll double check but it looks like you're right. If you are I'll refactor to use `Prim.RowList` for the "inner" list type. 

I think we still need `RowListI` for the 'outer' list of the data schema though.

Thanks for taking a look at this!